### PR TITLE
Update 2020_11_14-pkt-community.md

### PR DESCRIPTION
**Pull request by josh-berger**
Please find all of the info and assets for Milestone #1

### CODE REVIEW

**Comment by cjdelisle** [comment]
I had done a preliminary examination of this and suggested to Josh that he change one of the articles and one of the videos, which he did. Now I think it checks all of the boxes as per the requirements but we need to review it.

**Review by cjdelisle — APPROVED** [review]

**Review by backupbrain — COMMENTED** [review]
Nice and clean. Everything looks great. Nice to see PKT explained in the wild.

**Review by backupbrain — APPROVED** [review]

**Review by benhylau — APPROVED** [review]

File: projects/2020_11_14-pkt-community.md [hunk 1]
```diff
@@ -134,6 +134,22 @@ in this project.
...
+* VIDEO - 1 of 5 completed: https://youtu.be/_EeWHeY-omQ
+* TECH_ARTICLE - 2 of 6:
+** https://www.cryptopolitan.com/how-the-network-steward-ensures-funding-for-the-build-out-of-decentralized-internet-infrastructure-and-open-source-technology/
+** https://www.cryptopolitan.com/pkt-network-packet-priority-creates-a-more-economical-infrastructure-that-will-help-maximise-data-transmission/
+* ARTICLE - 2 of 6 completed:
+** https://docs.google.com/document/d/14G4sbUgrpuNjd9k0KrO0l10-9hnyVJLk/edit
+** https://securityonline.info/all-you-need-to-know-about-pkt-cryptocurrency/
+* GRAPHICS - 8 of 28 completed: https://drive.google.com/drive/folders/1gI_MiqkxkJZSsLobtbIddKWsB-izvA9d?usp=sharing
+* WWW_DOCS - launched with basic documentation written: https://docs.pkt.cash/en/latest/
+* WWW_BLOG - launched: http://crypto.pkt.cash/
```
Comment (neilalexander) [inline projects/2020_11_14-pkt-community.md:148]: 404?

**Review by cjdelisle — CHANGES_REQUESTED** [review]
Blog contains not a single article

File: projects/2020_11_14-pkt-community.md (hunk 1 above)
Comment (cjdelisle) [reply projects/2020_11_14-pkt-community.md:148]: Good call, I just changed my review until this is fixed.

**Comment by cjdelisle** [comment]
Ok, blog now contains articles and seems to be a process to add more, so I'm going to accept this.

**Review by cjdelisle — APPROVED** [review]

**Review by Arceliar — APPROVED** [review]
Some things about the production and editing in the first video seem a little off to me, but most of the subsequent videos (in an "interview" style) are much better I think. The audio quality could be better, I think the mic is picking up a little echo from the room (this is noticeably improved in the "tutorial" style videos). That said, overall, this looks good to me.

**Pull request merged**